pkgs/by-name/ai/aicommit2/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nodejs,
+  pnpm,
+  makeWrapper,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "aicommit2";
+  version = "1.12.3";
+
+  src = fetchFromGitHub {
+    owner = "tak-bro";
+    repo = "aicommit2";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-WKegxSnw7YgSa0H0pxhQws9MgYa08x3F/Q5MIePnTY4=";
+  };
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-Bdwskm81i6dleC/w/zK5iKbMIwBUhCp36sll+Ei5Wd4=";
```

Comment:
```suggestion
    hash = "sha256-NQQmk3ZYpHUCgqc89BmBEJKbh/c/Lvv+yz+Gqa+in30=";
```

Reply:
Please rebase and regenerate hashes.

See https://github.com/NixOS/nixpkgs/pull/349093#issuecomment-2421873637